http/sys_seal.go

```diff
@@ -155,16 +155,10 @@ func handleSysSealStatusRaw(core *vault.Core, w http.ResponseWriter, r *http.Req
 	var clusterName, clusterID string
 	if !sealed {
 		cluster, err := core.Cluster()
-
-		// Don't set the cluster details in the status when Vault is sealed
 		if err != nil {
 			respondError(w, http.StatusInternalServerError, err)
 			return
 		}
-		if cluster == nil {
```

Comment:
It's not a bad check to leave in just in case something changes, just put an actual error message in there. :-)